exporter/clients.go

```diff
@@ -1,12 +1,13 @@
 package exporter
 
 import (
-	"regexp"
-	"strings"
-
 	"github.com/gomodule/redigo/redis"
 	"github.com/prometheus/client_golang/prometheus"
 	log "github.com/sirupsen/logrus"
+	"regexp"
```

Comment:
Can you fix the sorting again? stdlibs first, then a blank line, then 3rd party imports.